apparelo/apparelo/doctype/label_fusing/label_fusing.py

```diff
@@ -64,26 +69,54 @@ def create_boms(self, input_item_names, variants, attribute_set,item_size,colour
 				boms.append(existing_bom)
 		return boms, input_item_list
 def create_item_template(self):
-	if not frappe.db.exists("Item", self.item+" Labeled Cloth"):
-		frappe.get_doc({
-			"doctype": "Item",
-			"item_code": self.item+" Labeled Cloth",
-			"item_name": self.item+" Labeled Cloth",
-			"description":self.item+" Labeled Cloth",
-			"item_group": "Sub Assemblies",
-			"stock_uom" : "Nos",
-			"has_variants" : "1",
-			"variant_based_on" : "Item Attribute",
-			"is_sub_contracted_item": "1",
-			"attributes" : [
-				{
-					"attribute" : "Apparelo Colour"
-				},
-				{
-					"attribute" : "Part"
-				},
-				{
-					"attribute" : "Apparelo Size"
-				}
-			]
-		}).save()
\ No newline at end of file
+	if self.based_on_style == 0:
```

Comment:
If only the `attributes` array changes based on the condition, then why to duplicate the whole object?